action-setup

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env bash
```

Comment:
This file replaces code that was previously hardcoded in `action.yml` and allows that code to be shared with `verify/action.yml`.